src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java

```diff
@@ -17,12 +17,12 @@
 
 @SpringBootApplication
 public class CompanyAccountsApplication implements WebMvcConfigurer {
+    public static final String APPLICATION_NAME_SPACE = "company-accounts.api.ch.gov.uk";
```

Comment:
Good idea, I'm sick off seeing the same string duplicated in each instance of where we defined a logger. :+1: